ml-proto/spec/kernel.ml

```diff
@@ -50,16 +50,16 @@ struct
            | ReinterpretInt
 end
 
-module Int32Op = IntOp ()
-module Int64Op = IntOp ()
-module Float32Op = FloatOp ()
-module Float64Op = FloatOp ()
+module I32Op = IntOp ()
+module I64Op = IntOp ()
+module F32Op = FloatOp ()
+module F64Op = FloatOp ()
```

Comment:
I'm having difficulty understanding how this code works. How would you describe what `I32Op` is here?